config/install/user.role.site_admin.yml

```diff
@@ -138,20 +410,65 @@ permissions:
   - 'use text format admin_text'
   - 'use text format rich_text'
   - 'use text format summary_text'
+  - 'view aboriginal_honour_roll revisions'
   - 'view alert revisions'
+  - 'view all media revisions'
   - 'view all revisions'
+  - 'view all scheduled transitions'
   - 'view any unpublished content'
   - 'view any webform submission'
+  - 'view event revisions'
+  - 'view field_business_contact_number'
+  - 'view field_business_name'
+  - 'view field_notes'
+  - 'view fv_recommendation revisions'
+  - 'view grant revisions'
+  - 'view igem_recommendation revisions'
   - 'view image_gallery revisions'
   - 'view landing_page revisions'
   - 'view landing_page revisions'
   - 'view latest version'
   - 'view moderation states'
   - 'view news revisions'
   - 'view note revisions'
+  - 'view own field_notes'
   - 'view own unpublished content'
   - 'view own webform submission'
   - 'view page revisions'
+  - 'view profile revisions'
+  - 'view project_infrastructure revisions'
+  - 'view publication revisions'
+  - 'view publication_page revisions'
+  - 'view scheduled transitions node aboriginal_honour_roll'
+  - 'view scheduled transitions node alert'
+  - 'view scheduled transitions node det_150_content'
+  - 'view scheduled transitions node event'
+  - 'view scheduled transitions node fv_recommendation'
+  - 'view scheduled transitions node grant'
+  - 'view scheduled transitions node igem_recommendation'
+  - 'view scheduled transitions node landing_page'
+  - 'view scheduled transitions node media_buying_for_vic'
+  - 'view scheduled transitions node news'
+  - 'view scheduled transitions node page'
+  - 'view scheduled transitions node profile'
+  - 'view scheduled transitions node project_infrastructure'
+  - 'view scheduled transitions node publication'
+  - 'view scheduled transitions node publication_page'
+  - 'view scheduled transitions node sr_profile'
+  - 'view scheduled transitions node test'
+  - 'view scheduled transitions node translated_page'
+  - 'view scheduled transitions node vada_profile'
+  - 'view scheduled transitions node vgso_profile'
+  - 'view scheduled update entities'
   - 'view section_page revisions'
+  - 'view sr_profile revisions'
   - 'view the administration theme'
+  - 'view tide_site preview links'
   - 'view timeline revisions'
+  - 'view translated_page revisions'
+  - 'view translated_vaccine_page revisions'
+  - 'view unpublished share link token entities'
+  - 'view vada_profile revisions'
+  - 'view vada_profile revisions'
+  - 'view vdrp_profile revisions'
+  - 'view vgso_profile revisions'
```

Comment:
please add the permissions that are needed. I can see there are lots of permissions which are only content-vic specific. Just add the permissions needed related to this PR please